android/titanium/src/java/org/appcelerator/titanium/TiLaunchActivity.java

```diff
@@ -167,16 +167,16 @@ protected void onCreate(Bundle savedInstanceState)
 		super.onCreate(savedInstanceState);
 	}
 
-	@Override
-	protected void windowCreated(Bundle savedInstanceState)
+	public boolean isJSActivity()
 	{
-		super.windowCreated(savedInstanceState);
-		loadScript();
+		return false;
 	}
 
-	public boolean isJSActivity()
+	@Override
+	protected void onResume()
 	{
-		return false;
+		super.onResume();
+		loadScript();
```

Comment:
@jquick-axway Any thoughts on moving `loadScript()` over to `onResume()`?

The reason for this is due to `handleOpen(options)` occurring before the RootActivity has resumed. Which prevents the transition from happening.

Reply:
I'm open to the idea. It should reduce app startup stutter as well. Let's test it out.

I think the only thing we lose is `Activity.onCreate` callback support for the root activity, but that's not useful. You're also less likely to receive `newintent` event from the root activity, but everyone should be reading the launch intent on app startup anyways and not depend on this behavior. We might have to worry about assumptions made after the `windowCreated()` call in the `TiBaseActivity.onCreate()`. We might be okay, but that's the part we need to safe-guard.